modular_skyrat/master_files/code/_globalvars/maint_loot_oddity.dm

```diff
@@ -65,5 +58,6 @@ GLOBAL_LIST_INIT(oddity_loot, list(//oddity: strange or crazy items
 	/obj/item/storage/toolbox/syndicate = 100,
 	/obj/item/storage/toolbox/infiltrator = 100,
 	/obj/item/toy/sprayoncan = 200,
-	/obj/item/warpwhistle = 5
+	/obj/item/warpwhistle = 5,
+	/obj/item/shadowcloak = 50
```

Comment:
Also, what does the shadowcloak do, again?
```suggestion
	/obj/item/shadowcloak = 50,
```

Reply:
the code does not like it if you put a comma on the last item in a list

Also, it lets you cloak to like, modsuit tiers, but can only recharge in absolute darkness, and you get like, 4s of invisiblity, it's a tf2 spy thing